### ISSUE TITLE
accept user generated sequence

### ISSUE THREAD

**Issue by katestange** [numberscope/frontscope#408]
Probably as a text file.  If users want to explore a sequence of their own, or extend the terms available on OEIS for a short sequence, they could create their own b-files and point Numberscope to those.

**Comment by gwhitney** [comment]
Yes, this is definitely a worthwhile idea. Is this something we ultimately want folded into Formulas as well? Or is this an argument that we won't actually ever be getting rid of different kinds of sequences, and that even if we can extend Formulas to allow OEIS sequences, we will still want to have both Formula and File sequences?

**Comment by katestange** [comment]
Good question.  One argument for keeping OEIS sequences as a sequence type separate from formula is just accessibility.

**Comment by gwhitney** [comment]
True. I can comfortably imagine a world in which we have Formula, OEIS, and File sequences, while still allowing multiple sequences to be visualized at once and supporting OEIS IDs in Formulas. But it will complicate the layout of the UI for multiple sequences-- can't just be a column of formula entry boxes.
